integration/kubernetes/k8s-ro-volume.bats

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2020 Ant Group
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+load "${BATS_TEST_DIRNAME}/../../lib/common.bash"
+
+setup() {
+	export KUBECONFIG="$HOME/.kube/config"
+	pod_name="test-readonly-volume"
+	container_name="busybox-ro-volume-container"
+	ro_volume_suffix="-tmp"
+	get_pod_config_dir
+}
+
+@test "Test readonly volume for pods" {
+	wait_time=20
```

Comment:
Since it is not using the `waitForProcess` function, I think wait_time and sleep_time aren't needed. Did I get it right?